### PR TITLE
Add argparser to stp.py

### PR DESCRIPTION
**Pull request by Erik-Lamers1**
Allow users to specify the width and height of the topology
Example
```
$ python stp.py --help
usage: This script will help you practise with spanning tree protocol topologies
       [-h] [--disable-banner] [-s] [-w {3,5,7,9,11}] [-H {3,5,7,9,11}]

optional arguments:
  -h, --help            show this help message and exit
  --disable-banner      Disable script banner
  -s, --skip-abbreviations
                        Do not ask abbreviation questions

Network size:
  Specify the network size parameters, these can be between 3 and 11 bridges
  and it has to be an odd number of bridges

  -w {3,5,7,9,11}, --width {3,5,7,9,11}
                        The width of the topology in bridges (default: 7)
  -H {3,5,7,9,11}, --height {3,5,7,9,11}
                        The height of the topology in bridges (default: 7)
```
This PR also add error reporting at the end of the practice run.

### CODE REVIEW

**Comment by Erik-Lamers1** [comment]
Check the last commits for the actual changes, the first commit is just my IDE applying code styles.

File: stp.py [hunk 1]
```diff
@@ -40,446 +38,494 @@
 rootW = -1
 rootH = -1
 
+
 # Classes
 class Edge:
-	classType = "Edge"
```
Comment (f13rce) [inline stp.py:45]: Please change it back to tabs, so I can see properly what changes you made :)

**Review by f13rce — CHANGES_REQUESTED** [review]

File: stp.py (hunk 1 above)
Comment (Erik-Lamers1) [reply stp.py:45]: That would be a very tedious process, as my code uses spaces by default. I've tried to revert 9d6fe268f7588686b16e282ab8ca5b5ea8070cd7 for you. But, that just gives indentation errors and conflicts. To see my actual changes you can look at the last 5 commits. Github review gives you the option to look at a certain commit range. 

The 9d6fe268f7588686b16e282ab8ca5b5ea8070cd7 commit changes no functionality it just auto reformats the file based on Pycharms code styles.

Comment (f13rce) [reply stp.py:45]: I've manually inspected the commits and I'll accept the changes - PR looks fine. I can live with the change from tabs to spaces.

**Review by f13rce — APPROVED** [review]

**Pull request merged**